app-admin/mkosi/mkosi-v10.ebuild

```diff
@@ -0,0 +1,25 @@
+# Copyright 2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{8..10} )
+
+DESCRIPTION="Build Bespoke OS Images"
+HOMEPAGE="https://github.com/systemd/mkosi"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS=""
+RDEPEND="sys-apps/portage dev-vcs/git dev-util/debootstrap"
```

Comment:
You can drop the `sys-app/portage` RDEPEND. Also maybe consider using the `optfeature` eclass for `dev-util/bootstrap`. I also think that mkosi requires systemd? At least that is what [an old comment of me on my mkosi build indicates](https://gitlab.com/Flow/flow-s-ebuilds/-/blob/a63c46ef2e4dd30100e63c6119c45e17f88609f4/app-admin/mkosi/mkosi-9999.ebuild#L28-29).

Reply:
If Portage is really needed, I would keep it there. IIRC mkosi directly calls `emerge`.